etc/config.sample.toml

```diff
@@ -95,6 +95,35 @@ reporting-disabled = false
   # log any sensitive data contained within a query.
   # query-log-enabled = true
 
+  # Settings for the TSM engine
```

Comment:
just `TSM` on purpose since it's in the config. My guess is even if we go with version 2 of the tsm format, we'll still have these configs.